src/ResourceManager/Dns/Commands.Dns/Zones/SetAzureDnsZone.cs

```diff
@@ -76,12 +76,21 @@ public override void ExecuteCmdlet()
                 zoneToUpdate.Name = zoneToUpdate.Name.TrimEnd('.');
                 this.WriteWarning(string.Format("Modifying zone name to remove terminating '.'.  Zone name used is \"{0}\".", zoneToUpdate.Name));
             }
+            ConfirmAction(
+                true,
```

Comment:
Same comment.  Call the overload with no Force parameter if no force parameter is present on the cmdlet